CHANGES.md

```diff
@@ -1,3 +1,16 @@
+## 1.3.3 2017-12-01
+
+:rotating_light: Bugfix extravanganza alert! :rotating_light:
+
+This release is an exciting one! We received a lot of contributions from our awesome Linkerd community. Special thanks
+to @sgrankin and @carloszuluaga just to name a few. Checkout our recent blog post for the full list of everyone that contributed
```

Comment:
- [ ] can we add a comma after "carloszuluaga" here? like "Special thanks to @sgrankin and @carloszuluaga, just to name a few."
- [ ] can we turn the GitHub usernames into Markdown links?
- [ ] once the blog post is live, can this link to it?